src/main/resources/default/taglib/w/autocompleteMultiSelect.html.pasta

```diff
@@ -3,7 +3,6 @@
 <i:arg name="name" type="String"/>
 <i:arg name="type" type="String" default=""/>
 <i:arg name="serviceUri" type="String"/>
-<i:arg name="searchKey" type="String" default=""/>
```

Comment:
"breaking change", because users have to remove parameter from their call. should I increment major-version?

Reply:
yes

Reply:
if we'd release a major - drop the two recently introduced @Deprecated (sonarlint shows them)

Reply:
ok, should I add that to this PR or create a separate PR for that?